contracts/masset/platform-integrations/AaveIntegration.sol

```diff
@@ -115,6 +115,58 @@ contract AaveIntegration is InitializableAbstractIntegration {
         return _checkBalance(aToken);
     }
 
+    /**
+     * @dev Move aTokens from Aave v1 to v2
+     *      Migrates from Aave v1 to Aave v2
+     * @param _bAssets     Array of bAsset addresses
+     * @param _newATokens  Address of newAToken addresses
+     * @param _platformAddress  Address of the platform
+     */
+    function migrate(address[] calldata _bAssets, address[] calldata _newATokens, address _platformAddress)
+    external
```

Comment:
Indent plz. If you download the `solidity` plugin for vsCode it should flash this up as a warning

Reply:
Sorted. Am a vim user but running things through `solc`, `solhint` and `solium` on save now.